Sources/Plasma/NucleusLib/pnNetBase/pnNbError.h

```diff
@@ -112,12 +112,9 @@ enum ENetError {
     // Be sure to add strings for additional error codes in pnNbError.cpp
     // (don't worry, the compiler will tell you if one is missing ;)
     kNumNetErrors,
-
-    // Net messages require ENetError to be sizeof(uint32_t)
-    kNetErrorForceDword             = (uint32_t)(-1)
 };
 
-//COMPILER_ASSERT_HEADER(pnNbError, sizeof(ENetError) == sizeof(uint32_t));
+static_assert(sizeof(ENetError) == sizeof(int32_t), "ENetError must be sizeof uint32_t");
```

Comment:
```suggestion
static_assert(sizeof(ENetError) == sizeof(int32_t), "ENetError must be sizeof int32_t");
```

Doesn't technically make a difference because both types are the same size, but better to be consistent 🙂

Reply:
Amended the commit so the type and the static assert message are aligned. Thanks!